src/search.njk

```diff
@@ -5,43 +5,104 @@ permalink: /search/
 {% extends 'layouts/base.njk' %}
 
 {% block content %}
-    <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
-        <symbol id="placeholder" viewBox="-4 -4 224 224">
-            {% include 'svg/placeholder.svg' %}
-        </symbol>
-    </svg>
-
-    <article class="search">
-        <h1>Search results: {{ '{{ searchQuery }}' }}</h1>
-        <p role="status" v-if="searchStatus">{{ '{{ searchStatus }}' }}</p>
-        <div class="news-grid" v-if="searchResults.length > 0">
-          <div v-for="item in searchResults" :key="item.id" class="api-content">
-              <article class="news-item">
-                  <figure v-if="item.category==='views'" class="preview-media">
-                      <svg class="logo news-item-img">
-                          <use xlink:href="#placeholder" />
-                      </svg>
-                  </figure>
-
-                  <h2 class="h3">
-                      <a :href="item.href">
-                          {{ '{{ item.title }}' }}
-                      </a>
-                  </h2>
-
-                  <div v-if="item.category==='views'" class="author">
-                      {{ '{{ item.author }}' }}
-                  </div>
-
-                  <div v-if="item.dateTime" class="date">
-                      <time :datetime="item.dateTime" class="dt-published">{{ '{{ item.dateTime }}' }}</time>
-                  </div>
-
-                  <div v-html="item.excerpt" class="preview-content" />
-              </article>
-          </div>
-      </div>
-    </article>
+	<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+		<symbol id="placeholder" viewBox="-4 -4 224 224">
+			{% include 'svg/placeholder.svg' %}
+		</symbol>
+		<symbol id="previous" width="14" height="24" viewBox="0 0 14 24">
+			{% include 'svg/prev.svg' %}
+		</symbol>
+		<symbol id="next" width="14" height="24" viewBox="0 0 14 24">
+			{% include 'svg/next.svg' %}
+		</symbol>
+	</svg>
+
+	<article class="search">
+		<h1>Search results: "{{ '{{ searchQuery }}' }}"</h1>
```

Comment:
This should use curly [double quotes](https://practicaltypography.com/straight-and-curly-quotes.html) `“”`

The same should be done for the _Tag_ page